pkg/api/v1/types.go

```diff
@@ -1229,6 +1229,9 @@ type PodSpec struct {
 	// If this option is set, the ports that will be used must be specified.
 	// Default to false.
 	HostNetwork bool `json:"hostNetwork,omitempty"`
+	// Uses the host's pid namespace.
```

Comment:
Use the host's PID namespace.  (rather than "Uses")

Please make the comment for HostNetwork consistent with this, also: Use the host's network namespace. (rather than "Uses")